README.md

```diff
@@ -11,8 +13,9 @@ Include the `highlight.js` script package in your webpage or node app, load this
 If you're not using a build system and just want to embed this in your webpage:
 
 ```html
-<script type="text/javascript" src="/path/to/highlight.pack.js"></script>
-<script type="text/javascript" src="/path/to/highlightjs-curl/curl.js"></script>
+<link rel="stylesheet" href="/path/to/highlight-styles/default.css">
+<script type="text/javascript" src="/path/to/highlight.min.js"></script>
+<script type="text/javascript" src="/path/to/highlightjs-curl.min.js"></script>
 <script type="text/javascript">
     hljs.registerLanguage('curl', window.hljsDefineCurl);
```

Comment:
Don't need this either.

Reply:
ok @joshgoebel I decided to take a different approach. my original intent was to give one-stop instructions how to use this, i have users only using curl highlighting and was trying to keep it simple for them, but it doesn't make sense to chase a moving target as the updates on highlight.js are the authoritative way to do it no sense to try to rehash the existing docs and keep them in sync. I ripped it all out and just provided links to the existing docs.

Reply:
Ok.  Just now seeing this.  We can see how that works.  It's just a bit different - but I totally see your point.  _Though I wouldn't say we're a very fast moving target_, but we did rename the "start up" JS, so perhaps that's what you're referring to? ;-)

Reply:
well, a lot of the sample start up code I had was from way back in maybe 9 or earlier and wasn't in sync with the current doc. If it changes again then languages like this would have extra maintenance requirements. i prefer single source of truth, but i also want to make it real easy for users to grab and go. tradeoff. i could copy what we have on cypher and change it to curl and just reuse that if you think that's ok.

Reply:
> a lot of the sample start up code I had was from way back in maybe 9 or earlier 

Well, 9 is now "very, very old" (in my mind at least) - and was before we even had official 3rd party grammars. We plan to release 11 in the spring/summer. :)  It's expected these things *could* change every major release (yearly-ish), but honestly I don't foresee them changing again for a while.  The only major doc change from 10 -> 11 OTTOMH is updating the API call from `initOnLoad` whatever to the new `highlightAll`...

Although outside things - like using ES6 modules on the client-side might become more popular and require doc changes - but that's not something we're driving, just life in the tech world. :)

Lets go with the reference to core docs and see how it works out. :)